### ISSUE TITLE
celery: Filesystem transport does not support `celery.app.control`

### ISSUE THREAD

**Issue by pmrowla** [iterative/dvc-task#7]
Kombu FS broker does not include `fanout` support, so the celery `control` API cannot be used to monitor workers using the FS broker. We can work around this by using our own `proc` based wrapper to monitor our local workers, but it may be better to just add proper support for this upstream.

There is an existing but abandoned PR for this upstream that should be an OK starting point:

https://github.com/celery/kombu/pull/1280

**Comment by auvipy** [comment]
it is fixed in kombu master now

**Comment by pmrowla** [comment]
fixed by https://github.com/celery/kombu/pull/1499

**Comment by karajan1001** [comment]
> fixed by [celery/kombu#1499](https://github.com/celery/kombu/pull/1499)

I guess we will still need to wait for the newest release and upgrade to it before we close this.

**Comment by pmrowla** [comment]
We can track waiting for the release in separate PR's/issues when we have changes that are actually blocked by the release.
